internal/scan/scan.go

```diff
@@ -159,6 +160,7 @@ func NewScanner(client EthClient, contracts ...string) Scanner {
 
 // ScanEvents returns the ERC721 events between fromBlock and toBlock
 func (s scanner) ScanNewUniversalEvents(ctx context.Context, fromBlock, toBlock *big.Int) ([]model.ERC721UniversalContract, error) {
+	slog.Info("scanning universal events ", "fromBlock", fromBlock, "toBlock", toBlock)
```

Comment:
Can you do the same thing for this line here https://github.com/freeverseio/laos-universal-node/blob/fix/remove-some-logs/internal/scan/scan.go#L249 and move it at the top of `ScanEvents`, please? Thank you

Reply:
changed